docs/static/settings/configuration-wildcard-pipeline-id.asciidoc

```diff
@@ -14,6 +14,6 @@ xpack.management.pipeline.id: ["*logs", "*apache*", "tomcat_log"]
 
 In this example, `"*logs"` matches all IDs ending in `logs`. `"*apache*"` matches any IDs with `apache` in the name.
 
-Wildcard in pipeline IDs is available starting with Elasticsearch 7.10. Logstash can pick up new pipeline without a restart if the new pipeline ID matches the wildcard pattern.
+Wildcard in pipeline IDs is available starting with Elasticsearch 7.11. Logstash can pick up new pipeline without a restart if the new pipeline ID matches the wildcard pattern.
```

Comment:
@jonny-mcc, the sources you are mentioning are belong to Logstash and this statement says [Logstash index REST API](https://github.com/elastic/elasticsearch/pull/53350) is available in Elasticsearch v7.10, which is not about Logstash. Please give your suggestion if this statement is confusing.
```suggestion
Wildcard in pipeline IDs is available starting with Elasticsearch 7.10. Logstash can pick up new pipeline without a restart if the new pipeline ID matches the wildcard pattern.
```